lib/senders/email.js

```diff
@@ -100,6 +100,17 @@ module.exports = function (log) {
     return 'messageType=fxa-' + templateName + ', app=fxa'
   }
 
+  function isFF57(message) {
+    const uaBrowser = message.uaBrowser
+    const uaBrowserVersion = message.uaBrowserVersion
+    const uaOS = message.uaOS
+
+    // Display new FF57 logo from all FF57 originating browsers
+    return (uaBrowser === 'Firefox' && uaBrowserVersion >= '57') ||
+      (uaBrowser === 'Firefox Android' && uaBrowserVersion >= '57') ||
+      (uaBrowser === 'Firefox' && uaBrowserVersion >= '10.0' && uaOS === 'iOS')
```

Comment:
Doing `>=` with numeric strings here seems risky to me, will this work in all the cases that we care about?  For example, if both sides are a string then `'9.3' >= '10.0'` will be `true`.